README.md

```diff
@@ -1,24 +1,44 @@
 # Angular Course
 
-Repository for the courses _Angular Foundation_ and _Advanced Angular: What the «Tour of Heroes» didn't tell you_.
+Repository for the courses _Angular Foundation_ and _Advanced Angular_.
 
 ## Getting started
 
-The `main` branch contains one Angular project with `subprojects` for each exercise and solution. They can be found in the folder [projects](projects). The subproject [00-base](projects/00-base) is the starting point and is based off the official [«Tour of Heroes» Angular Tutorial](https://angular.io/tutorial).
+This section will help you setup the Angular project that is used throughout the course.
+
+We will work with `subprojects` for each exercise and solution. They can be found in the folder [projects](projects).
+The subproject [00-base](projects/00-base) is your starting point and is based off the official [«Tour of Heroes» Angular Tutorial](https://angular.io/tutorial).
+
+### Prerequisites
+* [Node.js](https://nodejs.org/en/) installed (LTS version)
+* IDEA or Editor of your choice (e.g. [IntelliJ IDEA](https://www.jetbrains.com/idea/))
+* Terminal for running CLI commands
+
+### Instructions
+
+#### Install Angular CLI
+Open a terminal (if you're using an IDEA, you can open an integrated terminal) and run the following command:
```

Comment:
**suggestion:** I would leave that addition out, I don’t consider it very relevant and most IDEs/editors have an integrated terminal

```suggestion
Open a terminal and run the following command:
```